Cargo.toml

```diff
@@ -140,9 +140,6 @@ exclude = [
     "explorer",
     "contracts",
     "nym-wallet",
-    "nym-connect/mobile/src-tauri",
-    "nym-connect/desktop",
-    "nym-vpn/ui/src-tauri",
```

Comment:
why are we removing `nym-vpn`? why was it even there?

Reply:
old leftover, cargo probably doesn't check the existence of it